ui/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:8
```

Comment:
Please updates README to describe a procedure how to run the TokenBridge UI within the docker container.

Reply:
I have added a `ui/docker-compose.yml` and a step in the readme.